src/main/kotlin/org/rust/ide/actions/diagnostic/CreateNewGithubIssue.kt

```diff
@@ -44,7 +45,10 @@ class CreateNewGithubIssue : DumbAwareAction() {
             ?.displayText
         val ideNameAndVersion = ideNameAndVersion
         val os = SystemInfo.getOsNameAndVersion()
-        val macroEngine = project.rustSettings.macroExpansionEngine.name.toLowerCase()
+        val macroEngine = when (project.rustSettings.macroExpansionEngine) {
```

Comment:
nit: `macroEngine` name is a little bit confusing now. Maybe `macroExpansionEnabled` is better?